sentry/src/test/java/io/sentry/GsonSerializerTest.kt

```diff
@@ -486,6 +494,8 @@ class GsonSerializerTest {
         assertNotNull(transaction.timestamp)
         assertNotNull(transaction.contexts)
         assertNotNull(transaction.contexts.trace)
+        assertEquals("data-value", transaction.getData("string-data-key"))
```

Comment:
add a test setting a `Span#setData` too, right now only `Transaction` is being tested